src/figuresClasses.ts

```diff
@@ -1,11 +1,74 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: string = 'triangle',
+  ) {
```

Comment:
[CODE KNOWLEDGE] - TypeScript implicitly creates class properties from constructor parameters with access modifiers. You don't need to assign them manually as they are already available as object fields. You could simplify the constructor by removing the assignments and just declaring the parameters with their access modifiers.